Editor/UI/EditorWindows/AvatarLoaderEditor/AvatarLoaderEditor.cs

```diff
@@ -119,6 +118,11 @@ private void Completed(object sender, CompletionEventArgs args)
             }
             var paramHash = AvatarCache.GetAvatarConfigurationHash(avatarLoaderSettings.AvatarConfig);
             var path = $"{DirectoryUtility.GetRelativeProjectPath(args.Avatar.name, paramHash)}/{args.Avatar.name}";
+            if (!AvatarLoaderSettings.LoadSettings().AvatarCachingEnabled)
+            {
+                Debug.LogWarning("Enable Avatar Caching to generate a prefab in the project folder.");
+                return;
+            }
```

Comment:
This blocks it completely, so that it won't be created to the scene as well. Do we want it like that. Also should we use SDKLogger here